linux/vhba_hot_add_remove/freebsd_create_disk_partition.yml

```diff
@@ -46,20 +137,17 @@
   ansible.builtin.set_fact:
     partition_device_path: "/dev/{{ partition_name }}"
 
-- name: "Create {{ partition_fstype }} filesystem on partition {{ partition_device_path }}"
-  ansible.builtin.command: "newfs -U {{ partition_device_path }}"
+- name: "Create and check {{ partition_fstype }} filesystem on partition {{ partition_device_path }}"
+  ansible.builtin.shell: "newfs -EU {{ partition_device_path }} && sleep 5 && fstyp {{ partition_device_path }}"
   delegate_to: "{{ vm_guest_ip }}"
   ignore_errors: true
   register: partition_result
-  when: partition_fstype == "zfs"
-
-- name: "Create {{ partition_fstype }} filesystem on partition {{ partition_device_path }}"
-  community.general.filesystem:
-    fstype: "{{ partition_fstype }}"
-    dev: "{{ partition_device_path }}"
-  delegate_to: "{{ vm_guest_ip }}"
-  register: partition_result
-  when: partition_fstype == "ufs"
+  retries: 5
+  delay: 2
+  until:
+    - partition_result is defined
+    - partition_result.rc is defined
+    - partition_result.rc == 0
```

Comment:
The subsequent task run_iozone_taks.sh will report failure if not succeed after reries 5 times

Reply:
If `partition_result.rc` != 0, here can directly fail the test and report error about newfs failed. It can help us to address the problem as early as possible, no need to wait for iozone failure which is not the root cause failure.